src/state/stores.js

```diff
@@ -1,3 +1,4 @@
 import { writable } from "svelte/store";
 
 export const pageTitle = writable("");
+export const pageState = writable({});
```

Comment:
Out of curiosity, why did you decide to set up an empty store object, instead of populating them with properties and initial values? I'm used to seeing stores being more definitive, such as [this](https://svelte.dev/repl/1305d426ddef44578ae101bb1bc5cc8e?version=3.17.1). I think doing so is helpful because it tells us at first glance what states we're tracking across the app by looking at this store.

```suggestion
export const pageState = writable({
  itemType: "metrics",
  search: "",
  showExpired: "false",
  activeTab: "metrics",
  ...
});;
```

Reply:
Hmm, I wonder if this makes sense given that the applicable pageState depends on the page you're on?

Reply:
On reflection, I think we might want to do this, but let's wait a bit until we've added a few more pages (e.g. derived dataset documentation / searching). I think it will be more clear how to approach page state in a principled manner when we have a few more examples to work from. I think the current code is easy enough to understand.